assets/xml/objects/object_zo.xml

```diff
@@ -0,0 +1,88 @@
+<Root>
+    <File Name="object_zo" Segment="6">
+        <Skeleton Name="gZoraSkel" Type="Flex" LimbType="Standard" Offset="0xBFA8"/>
+
+        <Limb Name="gZoraPelvisLimb" LimbType="Standard" Offset="0xBE78"/>
+        <DList Name="gZoraPelvisDL" Offset="0x9040"/>
+
+        <!-- Left leg -->
+        <Limb Name="gZoraLeftThighsLimb" LimbType="Standard" Offset="0xBE84"/>
+        <DList Name="gZoraLeftThighsDL" Offset="0xB5F8"/>
+        <Limb Name="gZoraLeftShinLimb" LimbType="Standard" Offset="0xBE90"/>
+        <DList Name="gZoraLeftShinDL" Offset="0xB8D8"/>
+        <Limb Name="gZoraLeftFootLimb" LimbType="Standard" Offset="0xBE9C"/>
+        <DList Name="gZoraLeftFootDL" Offset="0xBBB0"/>
+
+        <!-- Right leg -->
+        <Limb Name="gZoraRightThighsLimb" LimbType="Standard" Offset="0xBEA8"/>
+        <DList Name="gZoraRightThighsDL" Offset="0xAD78"/>
+        <Limb Name="gZoraRightShinLimb" LimbType="Standard" Offset="0xBEB4"/>
+        <DList Name="gZoraRightShinDL" Offset="0xB058"/>
+        <Limb Name="gZoraRightFootLimb" LimbType="Standard" Offset="0xBEC0"/>
+        <DList Name="gZoraRightFootDL" Offset="0xB330"/>
+
+        <Limb Name="gZoraTorsoLimb" LimbType="Standard" Offset="0xBECC"/>
+        <DList Name="gZoraTorsoDL" Offset="0x91E8"/>
+
+        <!-- Left arm -->
+        <Limb Name="gZoraLeftArmLimb" LimbType="Standard" Offset="0xBED8"/>
+        <DList Name="gZoraLeftArmDL" Offset="0xA630"/>
+        <Limb Name="gZoraLeftForeArmLimb" LimbType="Standard" Offset="0xBEE4"/>
+        <DList Name="gZoraLeftForeArmDL" Offset="0xA890"/>
+        <Limb Name="gZoraLeftHandLimb" LimbType="Standard" Offset="0xBEF0"/>
+        <DList Name="gZoraLeftHandDL" Offset="0xAB70"/>
```

Comment:
Would upper arm and lower arm make better sense since instead of "forearm"?

Reply:
We usually use "forearm" (one word though).

Reply:
I think the proper words are "Arm" and "Forearm" (also, "Forearm" is one word, so I should change the capitalization).
But I could change them to upper arm and lower arm if it is preferred by more people.